distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Table_Helpers.enso

```diff
@@ -197,14 +198,15 @@ type Table_Column_Helper
        completely blank or have some blanks.
 
        Arguments:
-       - when_any: By default, only columns consisting of all blank cells are
-         selected. If set to `True`, columns with one or more blank values are
+       -TODO docs
+       - when: By default, only columns consisting of all blank cells are
+         selected. If set to Blank_Selector.Any_Cell_Blank, columns with one or more blank values are
          selected.
        - treat_nans_as_blank: If `True`, then `Number.nan` is considered as
          blank.
        - invert_selection: If `True`, then the selection is inverted.
-    select_blank_columns_helper : Boolean -> Boolean -> Boolean -> Vector
-    select_blank_columns_helper self when_any:Boolean treat_nans_as_blank:Boolean invert_selection:Boolean=False =
+    select_blank_columns_helper : Blank_Selector -> Boolean -> Boolean -> Vector
+    select_blank_columns_helper self (when:Blank_Selector = Blank_Selector.All_Cells_Blank) treat_nans_as_blank:Boolean invert_selection:Boolean=False =
```

Comment:
```suggestion
    select_blank_columns_helper self when:Blank_Selector treat_nans_as_blank:Boolean invert_selection:Boolean=False =
```

No need to have a default value here - this is an internal helper, never called by users.

In fact, I'd suggest to avoid the default - having a default makes it easier to forget to forward an argument and potentially lose a setting being propagated. Requiring it makes it IMO less likely to forget.